apps/system/js/statusbar.js

```diff
@@ -677,6 +678,12 @@ var StatusBar = {
       return;
     }
 
+    if (app instanceof AttentionWindow) {
+      this.element.classList.add('maximized');
+      this.element.classList.toggle('light', false);
```

Comment:
can we move these 2 lines in the `attentionopened` case and remove the `setAppearance` call? (and `AttentionWindow` reference) ?